sdk-wasm/src/lib.rs

```diff
@@ -24,14 +24,15 @@ fn map_async_jsvalue<E: std::error::Error>(
     })
 }
 
-#[wasm_bindgen]
-#[allow(non_snake_case)]
-/// Initialise console-error-panic-hook to improve debug output for panics.
-///
-/// Run once on initialisation.
-pub fn initPanicHook() {
-    console_error_panic_hook::set_once();
-}
+// removing since we have duplicate usage elsewhere
+// #[wasm_bindgen]
+// #[allow(non_snake_case)]
+// /// Initialise console-error-panic-hook to improve debug output for panics.
+// ///
+// /// Run once on initialisation.
+// pub fn initPanicHook() {
+//     console_error_panic_hook::set_once();
+// }
```

Comment:
Does having this duplicated break anything? If not I would suggest leaving it in so that it can be used if we need a "Kepler-only" wasm.

Otherwise, if it does break something then I think you could remove it entirely rather than commenting it out.

Reply:
I think it should be removed, the panic hook is really an app concern, not a library's

Reply:
But if this wasm is being used as a library in a pure ts/js app then there is no way for that app to set the panic hook. This is just a function that allows downstream consumers to set that for debugging purposes.

Reply:
oh right of course 🤦

Reply:
For this panic hook, ssx-rs also had a panic hook and it caused some kind of namespace collision error when bundling the wasm. I was unsure of how to make sure this panic hook was exported and used in wasm, so I left the upstream panic hook and removed this one. Open to changes here!